common.js

```diff
@@ -72,20 +76,14 @@ module.exports = {
     // * Prunes non-production node_modules (if opts.prune is set)
     // * Creates an asar (if opts.asar is set)
 
-    var tempParent = path.join(opts.tmpdir || os.tmpdir(), 'electron-packager', opts.platform + '-' + opts.arch)
-    var tempPath = path.join(tempParent, generateFinalBasename(opts))
+    var tempParent = opts.tmpdir === false ? null : path.join(opts.tmpdir || os.tmpdir(), 'electron-packager', opts.platform + '-' + opts.arch)
+    var tempPath = tempParent == null ? generateFinalPath(opts) : path.join(tempParent, generateFinalBasename(opts))
```

Comment:
We cannot use `==` for null in most cases. It is the reason why standard doesn't warn me about it (I suppose) and check was passed. Because` == null` means `=== null ||  === undefined`. In any case I will rewrite code to avoid "I'm not thrilled at having two ternary statements".